Makefile

```
@@ -17,39 +17,44 @@ pip: venv
 venv:
 	python3 -m venv --upgrade-deps .venv
 
-# the following commands can only be used when `make` was executed successfully
-# TODO issue-135: print a hint on the console to execute `make`
-jupyter:
+# General check for any tool in .venv/bin, based on the target name
+check-%:
+	@if [ ! -f .venv/bin/$* ]; then \
+		echo "You need to execute 'make' first to install dependencies"; \
```

Some of the dependencies are dev dependencies and not installed by `make` (see the pyproject.toml file)

Can you change the echo to make it more clear?

Something like this would be OK:

```suggestion
		echo "Tool '$*' not found in .venv/bin. Please run 'make' (or 'make install-dev') to install dependencies"; \
```

---

I updated the echo message.

I also changed the PR title to something more descriptive. I chose the title "Issue 135" to make it clear that I was working on the issue, since its currently not possible for us to directly assign ourself to an issue.

---

Thanks for the information, @KrasserKai , I was not aware that you cannot assign yourselves. I will update that in the guides 👍🏼